snuba/datasets/dataset.py

```diff
@@ -42,41 +34,31 @@ class Dataset(object):
     manipulate the lower layer objects.
     """
 
-    # TODO: Still not sure if the Dataset class really needs the actual Entity object
-    # or just the name.
     def __init__(self, *, default_entity: Entity) -> None:
-        # TODO: This is a convenience while we slowly migrate everything to Entities. This way
-        # every dataset can have a default entity which acts as a passthrough until we can
-        # migrate the datasets to proper entities.
+        # TODO: Right now there are no Datasets with more than one Entity. Once that changes,
+        # the entity accessor methods will need to be rewritten.
         self.__default_entity = default_entity
 
-    def get_entity(self, entity_name: Optional[str]) -> Entity:
+    def get_entity(self, entity_name: Optional[str] = None) -> Entity:
         return self.__default_entity
```

Comment:
Since we will will have to remove a concept of "default entity" when all the parts of snuba will be able to deal with a mullti-entity query, I would advise we keep `get_entity` and `get_default_entity` methods separate. It will be easier to hunt down the usages of `get_default_entity`, which can also be flagged as deprecated.